hbase-rest/pom.xml

```diff
@@ -160,7 +151,6 @@
     <dependency>
       <groupId>org.apache.hbase</groupId>
       <artifactId>hbase-protocol-shaded</artifactId>
-      <type>jar</type>
```

Comment:
This was just redundant, right?

Reply:
Yes, this was redundant.

From [Maven Coordinates](http://maven.apache.org/pom.html#Maven_Coordinates),
> When no packaging is declared, Maven assumes the packaging is the default: jar.